sdk/keyvault/azure-keyvault-administration/tests/test_backup_client.py

```diff
@@ -25,59 +25,57 @@ def create_key_client(self, vault_uri, **kwargs):
         credential = self.get_credential(KeyClient)
         return self.create_client_from_credential(KeyClient, credential=credential, vault_url=vault_uri, **kwargs )
 
-    @pytest.mark.parametrize("api_version", all_api_versions)
+    @pytest.mark.parametrize("api_version", only_default)
```

Comment:
Is there a particular reason we are using only the latest version of the service now?

Reply:
`use_managed_identity` support was only added in `7.5-preview.1`, so I wasn't sure if the feature would be enabled for earlier API versions. Also, since these tests now require a managed identity-enabled environment to record (I've been using Azure Cloud Shell), using the default version only makes this a little easier.